app/Libraries/ValidationErrors.php

```diff
@@ -41,7 +41,7 @@ public function add($column, $rawMessage, $params = null) : self
         }
         $rawMessage = $this->keyBase.$rawMessage;
 
-        $params['attribute'] = $column;
+        $params['attribute'] = trans($this->keyBase.$this->prefix.'.'.$column);
```

Comment:
- should check if translation exists
- group attribute/column translations so they're not mixed with actual error messages (inside `...{$this->prefix}.attributes...` or something).